trade-finance-manager-ui/server/controllers/case/underwriting/index.js

```diff
@@ -20,11 +21,21 @@ const { hasAmendmentInProgressDealStage, amendmentsInProgressByDeal } = require(
  * renders underwriting page with deal, tfm, dealId, user, leadUnderwriter, pricingAndRisk, underwriterManagersDecision
  */
 const getUnderwriterPage = async (req, res) => {
-  const dealId = req.params._id;
+  const dealId = req?.params?._id;
   const { user, userToken } = req.session;
 
+  if (!dealId || !user || !userToken) {
```

Comment:
Does our tfm middleware not handle having a session already (am assuming it's like portal) :) (so wouldn't need user token or user check here?)?